packages/calcite-components-angular/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "angular-workspace",
+  "private": true,
+  "scripts": {
+    "ng": "ng",
+    "prebuild": "./support/stencilDoubleBuildTypesWorkaround.sh && ts-node ./support/patchEventDetailTypeImports.ts",
+    "build": "ng build",
+    "clean": "rimraf dist node_modules .turbo .angular projects/component-library/dist"
+  },
+  "dependencies": {
+    "@angular/animations": "^16.2.0",
```

Comment:
Can we pin these deps and have renovate handle them?

Applies to all `package.json` files in this PR.

Reply:
Nice work @benelan!  What's your plan for updating Angular deps, e.g. each Calcite release? Mainly wondering how Calcite, Components and @arcgis/core can stay roughly in sync. For example, Angular 17 is being released next week.

Reply:
@jcfranco @benelan thoughts on this? Are you going with @latest, for each Calcite release? I don't know how 'renovate' is tied into the process. 

Also, will there be a `next` release of this that I can test on?

Reply:
Hi @andygup! Yeah renovate will submit a PR to bump the angular dependencies when new versions come out. 

And yes, a `next` release will be deployed automatically once there are "deployable" changes. I'll let you know when one goes out. Like the react wrapper, the angular wrapper's version will always match CC's to prevent confusion.